### PR TITLE
[PIN-2353] Adapted calls using tenant-management/process

### PR DESCRIPTION
**Pull request by bloodsky**

### CODE REVIEW

File: src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AgreementsApiServiceImpl.scala [hunk 1]
```diff
@@ -225,12 +222,14 @@ final case class AgreementsApiServiceImpl(
         .getTenant(agreement.consumerId)
         .zip(tenantManagementService.getTenant(agreement.producerId))
 
+      /*
       (consumerDescription, producerDescription) <- consumerTenant.selfcareId
         .fold(Future.successful(consumerTenant.id.toString()))(getDescription)
         .zip(producerTenant.selfcareId.fold(Future.successful(producerTenant.id.toString()))(getDescription))
+       */
```
Comment (TonioGela) [inline src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AgreementsApiServiceImpl.scala]: 🧹

File: src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PartyApiServiceImpl.scala [hunk 2]
```diff
@@ -133,7 +133,7 @@ final case class PartyApiServiceImpl(
       )
     )
   }
-
+  /*
```
Comment (TonioGela) [inline src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/PartyApiServiceImpl.scala]: 🧹

File: src/main/resources/interface-specification.yml [hunk 3]
```diff
@@ -2134,79 +2134,6 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
-  '/institutions/{tenantId}':
```
Comment (galales) [inline src/main/resources/interface-specification.yml:2137]: are we sure this endpoint is no more required?

Comment (TonioGela) [reply src/main/resources/interface-specification.yml:2137]: I think it got moved under /tenants/{tenantsId} at line 3054

**Review by TonioGela — APPROVED** [review]

**Review by galales — APPROVED** [review]

**Pull request merged**